tutorials/R/R-skills/Colorado-floods-data-visualization/NEON-Boulder-Flood-LiDAR-in-R/NEON-Boulder-Flood-LiDAR-in-R.Rmd

```diff
@@ -158,23 +174,23 @@ when plotting the second plot. To be able to see the first (hillshade) plot,
 through the second (DTM) plot, we also set a value between 0 (transparent) and 1 
 (not transparent) for the `alpha=` argument. 
 
-```{r plot-rasters}
+```{r plot-rasters, fig.cap=c(" Raster Plot of Four Mile Creek, Boulder County, Pre-Flood. This figure combines the DTM and hillshade raster objects into one plot.", "Raster Plot of Four Mile Creek, Boulder County, Post-Flood. This figure combines the DTM and hillshade raster objects into one plot.") }
 
 # plot Pre-flood w/ hillshade
-plot(DTMpre_hill,
-        col=grey(1:100/100),  # create a color ramp of grey colors for hillshade
+plot(DTM_pre_hillshade,
+        col=grey(1:90/100),  # create a color ramp of grey colors for hillshade
         legend=FALSE,         # no legend, we don't care about the grey of the hillshade
         main="Four Mile Canyon Creek, Boulder County\nPre-Flood",
         axes=FALSE)           # makes for a cleaner plot, if the coordinates aren't necessary
 
 plot(DTM_pre, 
         axes=FALSE,
-        alpha=0.5,   # sets how transparent the object will be (0=transparent, 1=not transparent)
-        add=T)  # add=TRUE (or T), add plot to the previous plotting frame
+        alpha=0.3,   # sets how transparent the object will be (0=transparent, 1=not transparent)
+        add=TRUE)  # add=TRUE (or T), add plot to the previous plotting frame
 
 # plot Post-flood w/ hillshade
 # note, no add=T in this code, so new plotting frame. 
-plot(DTMpost_hill,
+plot(DTM_post_hillshade,
         col=grey(1:100/100),  
         legend=FALSE,
         main="Four Mile Canyon Creek, Boulder County\nPost-Flood",
```

Comment:
Should we set alpha=0.3 for DTM_post?

Reply:
Thanks for input, @donal-at-NEON. Yes, I've updated the alpha value, and have addressed the other issues.